src/components/get_started/mainDescription.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div>
+    <el-divider class="divider" content-position="left">
+      Why We Created The Energy Dashboard
+    </el-divider>
+    <el-row>
+      The Energy Dashboard has been developed by the Oregon State University Sustainability Office to enable members of the OSU community to access, view, and analyze energy consumption trends.
+    </el-row>
+    <el-divider class="divider" content-position="left">
+      How To Use The "Map" Tab
+    </el-divider>
+      <mapTab />
+    <el-divider class="divider" content-position="left">
+      How To Use The "Building List" Tab
+    </el-divider>
+      <buildingListTab />
+  </div>
+</template>
+
+<script>
+import mapTab from '@/components/get_started/mapTab'
+import buildingListTab from '@/components/get_started/buildingListTab'
+
+export default {
+  components: {
+    mapTab,
+    buildingListTab
+  },
+  data () {
+    return {
+      pic1: '/images/solar.jpg'
+    }
+  }
+}
+</script>
+
+<style>
+.divider {
+  background-color: #D73F09;
+  margin-top: 40px;
+  margin-bottom: 40px;
```

Comment:
We may want to increase the font-size of the dividers in the future to make its internal text stand out a little more.